packages/ng-form/src/lib/utils/errors.ts

```diff
@@ -0,0 +1,15 @@
+import { ERROR_EXPECTED_MISSING_VALUE } from '../config'
+import { inject } from '@angular/core'
+
+export class FormException extends Error {
```

Comment:
visto che in alcuni casi gestivamo i controll con getValueOrError ho deciso di aggiungerlo anche qua, specifico però per FormException.

Avevo in mente 2 soluzioni alternative: o creare un'altra lib per gestire il getValueOrError o gestire in maniera alternativa. Entrambe le 2 soluzioni non mi ispiravano: tempo nel fare un'altra libreria eccessivo per poche righe di codice nel primo caso e valore perso dei controlli nel secondo. Per questo ho preferito importare direttamente qua la gestione degli errori per i form. Se non vi convince fatemi sapere